language/RELEASES.md

```diff
@@ -0,0 +1,72 @@
+* Fixed a compiler crash for an infinite loop with unreachable exits ([#7568](https://github.com/diem/diem/issues/7568)).
+* Fixed a compiler crash due to an unassigned local used in an equality comparison ([#7569](https://github.com/diem/diem/issues/7569)).
+* Fixed a compiler crash due to borrowing a divergent expression ([#7570](https://github.com/diem/diem/issues/7570)).
+* Fixed a compiler crash due to a missing constraint for references in the type checker ([#7573](https://github.com/diem/diem/issues/7573)).
+* Fixed a compiler crash related to expressions with short-circuiting ([#7574](https://github.com/diem/diem/issues/7574)).
+* Fixed an incorrect code generation bug that could occur when a function parameter is assigned a new value exactly once in the function ([#7370](https://github.com/diem/diem/pull/7370)).
+* Fixed the bytecode source map mapping from local names to indexes so that function parameters go before locals ([#7371](https://github.com/diem/diem/pull/7371)).
+* Fixed a compiler crash when a struct is assigned without specifying its fields ([#7385](https://github.com/diem/diem/issues/7385)).
+* Fixed a compiler crash when attempting to put a `spec` block inside a `spec` context ([#7387](https://github.com/diem/diem/issues/7387)).
+* An integer literal value that is too large for its declared type will no longer cause a compiler crash ([#7388](https://github.com/diem/diem/issues/7388)).
+* Fixed a compiler crash caused by incorrect number of type parameters in pack/unpack expressions ([#7401](https://github.com/diem/diem/pull/7401)).
+* Module names and module members are now restricted from starting with underscores (‘_’) , which also avoids a crash ([#7572](https://github.com/diem/diem/issues/7572)).
+* Prover specifications are now included in the compiler’s dependency ordering calculation ([#7960](https://github.com/diem/diem/pull/7960)).
+* Modified the compiler optimization to remove fall-through jumps so that loop headers are not coalesced, which improves the prover’s ability to handle loop specifications ([#8049](https://github.com/diem/diem/pull/8049)).
+
+## Command Line Interpreter (CLI)
+
+The Move CLI has been enhanced in several ways:
+
+* The CLI now supports safe module republishing with checks for breaking changes ([#6753](https://github.com/diem/diem/pull/6753)).
+* Added a new `doctor` command to detect inconsistencies in storage ([#6971](https://github.com/diem/diem/pull/6971), [#7010](https://github.com/diem/diem/pull/7010), and [#7013](https://github.com/diem/diem/pull/7013)).
+* The `publish` command’s `—-dry-run` option has been removed ([#6957](https://github.com/diem/diem/pull/6957)). Use the equivalent "check" command instead.
+* The `test` command has a new `--create` option to create test scaffolding ([#6969](https://github.com/diem/diem/pull/6969)).
+* The verbose output now includes the number of bytes written ([#7757](https://github.com/diem/diem/pull/7757)).
```

Comment:
```suggestion
* The verbose output -v` now includes the number of bytes written ([#7757](https://github.com/diem/diem/pull/7757)).
```

Reply:
I'll change it to: The verbose output with the `-v` option....